components/romio/src/ome/io/nio/PixelsService.java

```diff
@@ -639,6 +649,30 @@ protected int getSeries(Pixels pixels)
         }
     }
 
+    /**
+     * Retrieves the series for a given set of pixels.
+     * @param pixels Set of pixels to return the series for.
+     * @return The series as specified by the pixels parameters or
+     * <code>0</code> (the first series).
+     */
+    protected LutSource getLutSource(Map<String, String> params)
+    {
+        String lut = null;
+        for (Map.Entry<String, String> param : params.entrySet()) {
+            if (param.getKey().startsWith("lut")) {
```

Comment:
Definitely *not*. This was awaiting our storage decision. It will be converted to an annotation with a proper namespace.